web/html/src/manager/content-management/shared/business/filters.enum.ts

```diff
@@ -86,6 +96,11 @@ const filterMatchers: FilterMatcherEnumType = {
     text: t("later or equal"),
     longDescription: t("later or equal than"),
   },
+  GREATER: {
+    key: "greater",
+    text: t("later"),
+    longDescription: t("later than"),
```

Comment:
Hm, looking at this, shouldn't this be "greater" and "greater than"? Same goes for the existing text above...

I think we only used this for the patch issue date so far, where "later" made sense, but now it'd be better to change it is more general "greater"/"greater than".